components/Footer/index.js

```diff
@@ -47,14 +61,26 @@ const NavLink = styled.li`
   align-items: center;
   list-style-type: none;
   padding: 0px 5px;
+  color: ${(props) => (props.active ? "#00ff00" : "#95091b")};
+  cursor: pointer;
+  a {
+    color: #95091b;
+    text-decoration: none;
+    &:hover {
+      color: #099584;
+      font-size: 1.1em;
+      transform: scale(1.1);
+    }
```

Comment:
We'd usually want to avoid nested type selectors as part of styled components. You may want to outsource this to its own styled component.

Reply:
Yes, this is something that I will focus on after the chunkier functionality is all set.